sdk/js/index.js

```diff
@@ -1,8 +1,26 @@
+const { spawn } = require('child_process');
+
 const runners = {
-  "linux-bash-game-v0": (agent, formatAddress, gameData) => {
+  "linux-bash-game-v0": async (agentId, formatAddress, gameData) => {
     // presume we are running a node linux bash squad client
     // start a process identified in the game data
-    
+
+    // get the format from holochain 
+    const format = [formatAddress]
+    // pass the format into a new child process that takes in the format and starts the game (Formats not being passed yet)
+    const download = await spawn(gameData.cmd, gameData.options)
+    download.on('close', async () => {
+      const permissions = await spawn('chmod', ['u+x', `./${gameData.folder}/squad_launcher.sh`])
+      permissions.on('close', async () => {
+        const game = await spawn(`./${gameData.folder}/squad_launcher.sh`)
```

Comment:
I think we should be able to move all this functionality into `gameData.cmd`, and `gameData.options` 

For example cmd could be `curl https://github.com/SetMatchGames/smg-proof-of-concept/folder/install_and_run.sh | sh`

Reply:
This way we don't require the games include a `squad_launcher.sh` and give them more flexibility, for example games would more easily be portable to squad without those requirements.